mmv1/products/workstations/terraform.yaml

```diff
@@ -109,3 +109,19 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           workstation_cluster_name: "workstation-cluster"
           workstation_config_name: "workstation-config"
+  Workstation: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format:
+      [
+        "projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations/{{workstation_id}}",
+        "{{workstation_id}}",
```

Comment:
```suggestion
```

I don't think that this would work- the GET URL's `projects/{{project}}/locations/{{location}}/workstationClusters/{{workstation_cluster_id}}/workstationConfigs/{{workstation_config_id}}/workstations/{{workstation_id}}` which means the minimum set of values would be `{{location}}/{{workstation_cluster_id}}/{{workstation_config_id}}/{{workstation_id}}`

Reply:
We have a similar import format in the other cloud workstations too, what are your thoughts on those? I'll remove what you've suggested for the Workstation.

Reply:
Applied your suggested change: removed the line in the import format.

Reply:
Assuming they don't work (& by a static reading of the code I don't think they do), yeah, we should remove them.

Reply:
Removed them 👍🏼